TI-reports/2025/2025-Q3-Alpha-Omega.md

```diff
@@ -0,0 +1,76 @@
+
+We recently approved a grant to [ecosyste.ms](https://ecosyste.ms) to enable API authentication, limits, and metrics; contributor onboarding guidance, and a governance / sustainability plan.
+
+In July 2025, we approved funding through OpenRefactory to build out VEX automation tooling, with Apache Airflow as an engaged partner.
+
+## Recent Events / News / Blogs / Etc.
+
+* [Member Spotlight: AWS - Funding Open Source Security Empowerment](https://alpha-omega.dev/blog/member-spotlight-aws-funding-open-source-security-empowerment/), Henri Yandell, AWS
+* [OSTIF 2025 Alpha-Omega Partnership Updates and Roadmap](https://alpha-omega.dev/blog/ostif-2025-alpha-omega-partnership-updates-and-roadmap/), Helen Woeste, OSTIF
+* [Preventing Domain Resurrection Attacks](https://blog.pypi.org/posts/2025-08-18-preventing-domain-resurrections/) (Mike Fiedler, Python Software Foundation)
+* [Unmasking Phantom Dependencies with Software Bill-of-Materials as Ecosystem-Neutral Metadata](https://alpha-omega.dev/wp-content/uploads/sites/22/2025/08/Python-White-Paper-for-AO-3.pdf) (Seth Larson, Python Software Foundation)
+* [Funding Security in Open Source: Insights From FreeBSD's Audit Journey](https://www.youtube.com/watch?v=B7_0aKMs6zs) (Deb Goodkin, FreeBSD Foundation; Michael Winser, Alpha-Omega)
+* [Reviewing another year of security partnership between the PSF & Alpha-Omega](https://www.youtube.com/watch?v=ZbHzU72r3mY) (Seth Michael Larson, Python Software Foundation; Michael Winser, Alpha-Omega)
+
+Alpha-Omega Q3 Update: Will be published later this month at [alpha-omega.dev/resources/reports/](https://alpha-omega.dev/resources/reports/).
+
+## Upcoming Events
+
+* Alpha-Omega Q3 Virtual Roundtable on Open Source Sustainability
+* Alpha-Omega Public Meeting - October 1, 2025 -- **please join us - it's on the OpenSSF public calendar**
+
+## Objectives & Key Results
+
+| **Objective #1: Catalyze trustworthy and secure software, runtimes, and infrastructure for all the major open source ecosystems through staffing** | |
+|-|-|
+| **KR1.1**: Fund security improvements and initiatives for at least ten critical open source organizations by the end of 2025. | On Target |
+| **KR 1.2**: For each engagement, confirm progress toward improved security outcomes, evidenced through initial and/or follow-on assessments, monthly reporting, and periodic check-ins. | In Progress |
+| **KR 1.3**: Drive the organizations we work with to obtain security funding from at least one organization other than Alpha-Omega, targeting 33% by the end of 2025. | In Progress |
+| **KR 1.4**: Organize quarterly roundtables for at least 5 major ecosystems  to share information, build connections, and collaborate, resulting in at least one new project or joint publication started in 2025. | Started |
```

Comment:
I'm curious why this KR is still marked as Started at this point given this and past quarters' roundtables. Is the Started status referring mostly to the "new project or joint publication" part?